modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vmi-yaml-ssh_{context}"]
+= Accessing a virtual machine instance via SSH with YAML configurations
+
+You can enable a SSH connection to a virtual machine instance (VMI) without the need to run the `virtctl expose` command. When the YAML file for the VMI and the YAML file for the service are configured and applied, the service forwards the SSH traffic to the VMI.
+
+The following examples show the configurations for the VMI's YAML file and the service YAML file.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+* You must be in the same project as the VMI.
```

Comment:
1. The user doesn't have to be in the same project as the VMI; the service does :)
2. To ensure that - add the "namespace" entry in the metadata of both the VM's and Service's manifest yamls.

Reply:
1) This sentence (You must be in the same project...) is used universally in other assembly modules of our CNV doc set. If a change is to be made, it should be made universally across the doc set. Adding @ctomasko@redhat.com and @aspauldi@redhat.com for awareness and to evaluate in a doc team meeting whether this change needs to be made universally across our doc set.

2) Deferring this input to @phoracek, @myakove, and @alonsadan@redhat.com.

Reply:
Maybe, but the common way of creating a namespaced resource in a specific project is by adding the "metadata.namespace" entry in its manifest YAML. In this case, which explicitly deals with creating the service via YAML, I think it should be part of the YAML.

Reply:
Added namespace to both yaml manifests.